server/percona_auth_test.go

```diff
@@ -0,0 +1,41 @@
+package server
+
+import (
+	"testing"
+	"time"
+
+	"github.com/brianvoe/gofakeit"
+	serverClient "github.com/percona/pmm/api/serverpb/json/client"
+	"github.com/percona/pmm/api/serverpb/json/client/server"
+	"github.com/stretchr/testify/require"
+
+	pmmapitests "github.com/Percona-Lab/pmm-api-tests"
+)
+
+func TestPerconaAuth(t *testing.T) {
+	client := serverClient.Default.Server
+	login := gofakeit.Email()
+	password := "Password12345"
```

Comment:
same as this one https://github.com/percona/pmm-managed/pull/420#discussion_r456396635
consider using gofakeit for password generation

Reply:
Done